src/components/StokesAnalysis/StokesAnalysisComponent.tsx

```diff
@@ -139,6 +139,20 @@ export class StokesAnalysisComponent extends React.Component<WidgetProps> {
         this.widgetStore.clearScatterPlotXYBounds();
     };
 
+    // true: red->blue, false: blue->red
+    private getColorMapOrder(frame: FrameStore): boolean {    
+        if (frame && frame.channelInfo && frame.channelInfo.fromWCS && frame.channelInfo.channelType && !isNaN(frame.channelInfo.delta)) {
+            let CTYPE = frame.spectralType !== null? frame.spectralType : frame.channelInfo.channelType.code;
+            if (CTYPE === SpectralType.CHANNEL) {
+                CTYPE = frame.channelInfo.channelType.code;
+            }
+            const inGroup = SPECTRAL_COLORMAP_GROUP.includes(CTYPE);
```

Comment:
I think it would be much easier to not create this array, and just manaully check if it's `===` one of the three options. These are standardised options and will not change